sdk/src/test/java/com/mapbox/maps/renderer/MapboxRendererTest.kt

```diff
@@ -3,11 +3,7 @@ package com.mapbox.maps.renderer
 import android.os.Handler
 import android.os.HandlerThread
 import android.os.Looper
-import com.mapbox.common.ShadowLogger
-import com.mapbox.maps.MapInterface
-import com.mapbox.maps.MapView
-import com.mapbox.maps.Size
-import com.mapbox.maps.Task
+import com.mapbox.maps.*
```

Comment:
maybe we need rule for AS for wildcard imports?

Reply:
we don't have it now so I see no issue in using it, we're doing this in dozens of other places. Let's revisit that separately in dedicated PR if we decide on it, WDYT?